core/src/main/java/org/codice/imaging/nitf/core/image/impl/RasterProductFormatUtilities.java

```diff
@@ -59,25 +66,42 @@ public RasterProductFormatUtilities() throws NitfFormatException {
     }
 
     /**
-        Unmarshal input stream to RPFs list.
-
-        @param inputStream the input stream to parse.
-        @throws JAXBException if the unmarshalling of the XML fails.
-    */
+     * Unmarshal input stream to RPFs list.
+     *
+     * @param inputStream the input stream to parse.
+     * @throws JAXBException if the unmarshalling of the XML fails.
+     */
     private void unmarshal(final InputStream inputStream) throws JAXBException {
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+        Document document = null;
+        try {
+            dbf.setFeature("http://xml.org/sax/features/external-general-entities", false);
+            dbf.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+            dbf.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
```

Comment:
I don't believe they make calls to those URLs, I think they are just the names of the features. That being said, that's why I included the `try/catch`, because I don't believe these features are absolutely necessary, but are nice to have.